src/main/java/TaskList.java

```diff
@@ -0,0 +1,77 @@
+import java.util.ArrayList;
+
+/**
+ * Stores and maintains the list of tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+    private int size;
+
+    public TaskList() {
+        tasks = new ArrayList<Task>();
+        size = 0;
+    }
+
+    /**
+     * Formats the current tasks into a String to be saved into a txt file.
+     * @return A String of all the current tasks
+     */
+    public String tasksToString() {
```

Comment:
a bit picky but overriding the toString() method might be a better idea as it is more integrated, i.e. you can System.out.println(myTaskList); instead of System.out.println(myTaskList.tasksToString());